### ISSUE TITLE
Incorrect relative description when r is much much larger than other letters

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/resistance-in-a-wire#129]
From phetsims/qa#92, @terracoda said

> The only description that I was not expecting was this one where there are three explicit "much much larger than"'s. All three sliders are different, and R is so huge it is off screen. I don't think have a better way to handle this edge case, so I think we have to go with "much much larger" across the equation phrase.

![](https://user-images.githubusercontent.com/1902116/36051781-ca4a1c94-0dc5-11e8-9e05-846a4c230226.png)

Thanks for the report @terracoda, it should say

"Size of letter R is much much larger than the size of letter rho, letter L, and letter A."

**Comment by terracoda** [comment]
@jessegreenberg, yes, that is what it should say. If the comparison is all the same, it is better not to repeat the phrases.

**Comment by jessegreenberg** [comment]
The problem is that rho, L, and A scales are not at comparable levels, but the pattern used needs to describe them as such.

**Comment by jessegreenberg** [comment]
Should be fixed in the above commit. Thanks @terracoda, closing.
